### PR TITLE
Updated README for schedule initial phase

### PR DESCRIPTION
**Pull request by brgnepal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
-# freshkite-picnic
-Freshkite picnic  project
+# Freshkite Picnic
+
+This repository contains the information related to the freshkite picnic
```
Comment (Esaimozhi3) [inline README.md:3]: This line is missing our mission for the picnic

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+which happens annually.
+
+# Schedule
+
+1. Meeting a common point - 6 AM
+2. Bus journey start      - 7 AM
```
Comment (Esaimozhi3) [inline README.md:13]: This schedule is missing lunch time.

**Review by Esaimozhi3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (brgnepal) [reply README.md:3]: Fixed

File: README.md (hunk 2 above)
Comment (brgnepal) [reply README.md:13]: Updated schedule

**Review by Esaimozhi3 — APPROVED** [review]

**Pull request merged**